### DISCUSSION TITLE
用\chapter*创建致谢后，试图通过\addcontentsline将其添加至目录时目录字体不正确

### DISCUSSION THREAD

**Discussion by AlexeyIVladimirov** [stone-zeng/fduthesis#284] [Q&A] [answered]
通过\addcontentsline{toc}{chapter}{后记与致谢}
\chapter*{后记与致谢}
试图创建一个致谢章节并将其添加至目录，此时发现目录字体不正确，和其它正式章在目录中的字体不同（呈现出来的效果像是加粗的宋体）
![image](https://user-images.githubusercontent.com/57180619/209982731-5f939e40-b46b-47f2-b324-4e43b9ad98fe.png)
如果使用\addcontentsline{toc}{chapter}{\sffamily 后记与致谢}
效果则类似于加粗后的黑体，依旧与其它正式章在目录中的字体不同
![image](https://user-images.githubusercontent.com/57180619/209982877-8c6d6aba-abbd-4aea-9254-9de2676f7569.png)
作为对比的其它正式章在目录中的字体：
![image](https://user-images.githubusercontent.com/57180619/209982937-a59e37fd-87e4-439c-93a1-d1e06fcd3b5e.png)

**Comment by sikouhjw** [comment, marked as answer]
![图片](https://user-images.githubusercontent.com/49770925/209993850-838a6872-de31-41ba-bffa-08e5a78b88fa.png)
`fduthesis` 内部是用 `\__fdu_chapter:V` 来实现你的需求的，按理说没提供接口代表用户不需要手动添加无编号的章节。

**Reply by AlexeyIVladimirov** [reply]
非常感谢，我改成
```
\ExplSyntaxOn
\__fdu_chapter:n{后记与致谢}
\ExplSyntaxOff
```
以后就一切正常了
